jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/spat/SpatTopology.java

```diff
@@ -61,17 +58,14 @@ public static Topology build(String spatOdeJsonTopic, String spatProcessedJsonTo
                     return deserializedRawSpat;
                 });
 
-        // Convert ODE SPaT to GeoJSON
-        // KStream<RsuIntersectionKey, ProcessedSpat> processedJsonSpatStream =
-        // validatedOdeSpatStream.transform(() -> new SpatProcessedJsonConverter() // change this converter to
-        // // something else NOT GEOJSON
-        // );
+        // Convert ODE SPaT to ProcessedSpat which is not GeoJSON
+        KStream<RsuIntersectionKey, ProcessedSpat> processedJsonSpatStream =
+                validatedOdeSpatStream.transform(() -> new SpatProcessedJsonConverter());
 
```

Comment:
suggestion: Do CTI 4501 validation on the ProcessedSpats here.

Reply:
I added this to the conversion class since we need access to the ode output.